trpcultivate_phenotypes/src/Plugin/TripalImporter/TripalCultivatePhenotypesTraitsImporter.php

```diff
@@ -168,7 +168,7 @@ public function configureValidators($form_values) {
     // - All header row cells are not empty.
     // @TODO: Uncomment the following code when the Headers validator has been
     //        updated and no longer uses the validate() method.
-    // $instance = $manager->createInstance('empty_cell');
+    // $instance = $manager->createInstance('empty_header_cell');
```

Comment:
This will still need to be `empty_cell` being passed to create instance but then it will be saved as `$validators['header-row']['empty_header_cell']` to indicate it's uniqueness.

```suggestion
    // $instance = $manager->createInstance('empty_cell');
```